tests/framework/ROM/TimeSeries/SyntheticHistory/TrainingData/generateARMA.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
Would this file and the two other .csv files go better in the tests folder?

Reply:
The RAVEN run uses this training data, so I think it makes sense to have it here (along with the generated data).

The other option would be to put each generation script and the CSVs in the RAVEN working dirs for each integration test. I don't know which I prefer; I like the grouped similarity of the analytic training data being in a central place.

Reply:
Oh I see, I think I like your first suggestion better, but also like having the analytic stuff in one location. I think making those changes would be better served in a different PR so no hold ups here.